Issues with Go 1.13

## Version of Singularity:

`3.4.0-128.g9250bec41` (current `master` when issue is filed.)

### Expected behavior

Singularity should be compatible with Go v1.13.

### Actual behavior

Singularity is not currently compatible with Go v1.13. For example:

```sh
$ make -C builddir/ vendor
make: Entering directory '/home/adam/go/src/github.com/sylabs/singularity/builddir'
 VENDOR
go: github.com/sylabs/scs-key-client@v0.3.0-0.20190509220229-bce3b050c4ec: invalid pseudo-version: version before v0.3.0 would have negative patch number
Makefile:560: recipe for target 'vendor' failed
make: *** [vendor] Error 1
make: Leaving directory '/home/adam/go/src/github.com/sylabs/singularity/builddir'
```

The source of the issue appears to be that the version of the `github.com/sylabs/scs-key-client` package in `go.mod` and `go.sum` is not valid. Go 1.13 added version validation ([ref](https://golang.org/doc/go1.13#modules)).

External projects using Go v1.13 that import `github.com/sylabs/singularity` will also experience this issue. It manifests itself similarly:

```sh
go: github.com/sylabs/singularity@v0.0.0-20190731162714-49ec4f26125f requires
	github.com/sylabs/scs-key-client@v0.3.0-0.20190509220229-bce3b050c4ec: invalid pseudo-version: version before v0.3.0 would have negative patch number
```

### Steps to reproduce behavior

Run `make -C builddir/ vendor` with Go v1.13 installed, or import the `github.com/sylabs/singularity` module from a project using Go v1.13.

--- a/go.mod
+++ b/go.mod
@@ -79,7 +79,7 @@ require (
 	github.com/stevvooe/resumable v0.0.0-20180830230917-22b14a53ba50 // indirect
 	github.com/sylabs/json-resp v0.5.0
 	github.com/sylabs/scs-build-client v0.0.4
-	github.com/sylabs/scs-key-client v0.3.0-0.20190509220229-bce3b050c4ec
+	github.com/sylabs/scs-key-client v0.3.1-0.20190509220229-bce3b050c4ec
 	github.com/sylabs/scs-library-client v0.4.2
 	github.com/sylabs/sif v1.0.8
 	github.com/syndtr/gocapability v0.0.0-20180223013746-33e07d32887e // indirect

